src/integrationTest/resources/simulations/__files/user-asmith_gen.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
Needed a new wiremock mapping for "Alex Smith" as one of the integration tests expects that value based on values in one of the test data builders (if I had changed the test data builder I likely would have broken other tests and ended up chasing my tail!)